manpages/pihole-FTL.conf.5

```diff
@@ -7,43 +7,143 @@ pihole-FTL.conf - FTL's config file
 
 /etc/pihole/pihole-FTL.conf will be read by \fBpihole-FTL(8)\fR on startup.
 .br
+For each setting the option shown first is the default.
+.br
 
-\fBSOCKET_LISTENING=localonly|all\fR
+\fBBLOCKINGMODE=IP|IP-AAAA-NODATA|NODATA|NXDOMAIN|NULL\fR
+.br
+    How should FTL reply to blocked queries?
+
+    IP             - Pi-hole's IPs for blocked domains
+
+    IP-AAAA-NODATA - Pi-hole's IP + NODATA-IPv6 for blocked domains
+
+    NODATA         - Using NODATA for blocked domains
+
+    NXDOMAIN       - NXDOMAIN for blocked domains
+
+    NULL           - Null IPs for blocked domains
+.br
+
+\fBCNAME_DEEP_INSPECT=true|false\fR
+.br
+    Use this option to disable deep CNAME inspection. This might be beneficial for very low-end devices.
+.br
+
+\fBBLOCK_ESNI=true|false\fR
+.br
+    Block requests to _esni.* sub-domains.
+.br
+
+\fBMAXLOGAGE=24.0\fR
 .br
-    Listen only for local socket connections or permit all connections
+    Up to how many hours of queries should be imported from the database and logs?
+.br
+    Maximum is 744 (31 days)
+.br
+
+\fBPRIVACYLEVEL=0|1|2|3|4\fR
+.br
+    Privacy level used to collect Pi-hole statistics.
+.br
+    0 - show everything
+.br
+    1 - hide domains
+.br
+    2 - hide domains and clients
+.br
+    3 - anonymous mode (hide everything)
+.br
+    4 - disable all statistics
 .br
 
-\fBQUERY_DISPLAY=yes|no\fR
+\fBIGNORE_LOCALHOST=no|yes\fR
 .br
-    Display all queries? Set to no to hide query display
+    Should FTL ignore queries coming from the local machine?
 .br
 
 \fBAAAA_QUERY_ANALYSIS=yes|no\fR
 .br
-    Allow FTL to analyze AAAA queries from pihole.log?
+    Should FTL analyze AAAA queries?
+.br
+
+\fBANALYZE_ONLY_A_AND_AAAA=false|true\fR
+.br
+    Should FTL only analyze A and AAAA queries?
+.br
+
+\fBSOCKET_LISTENING=localonly|all\fR
+.br
+    Listen only for local socket connections on the API port or permit all connections.
+.br
+
+\fBFTLPORT=4711\fR
+.br
+    On which port should FTL be listening?
 .br
 
 \fBRESOLVE_IPV6=yes|no\fR
 .br
-    Should FTL try to resolve IPv6 addresses to host names?
+    Should FTL try to resolve IPv6 addresses to hostnames?
 .br
 
 \fBRESOLVE_IPV4=yes|no\fR
 .br
-    Should FTL try to resolve IPv4 addresses to host names?
+    Should FTL try to resolve IPv4 addresses to hostnames?
 .br
 
-\fBMAXDBDAYS=365\fR
+\fBDELAY_STARTUP=0\fR
+.br
+    Time in seconds (between 0 and 300) to delay FTL startup.
+.br
+
+\fBNICE=-10\fR
+.br
+    Set the niceness of the Pi-hole FTL process.
+.br
+    Can be disabled altogether by setting a value of -999.
+.br
+
+\fBNAMES_FROM_NETDB=true|false\fR
+.br
+    Control whether FTL should use a fallback option and try to obtain client names from checking the network table.
+.br
+    E.g. IPv6 clients without a hostname will be compared via MAC address to known clients.
+.br
+
```

Comment:
```suggestion

\fBREFRESH_HOSTNAMES=IPV4|ALL|NONE\fR
.br
    Change how (and if) hourly PTR requests are made to check for changes in client and upstream server hostnames. The following options are available:
.br
    IPV4 = Do the hourly PTR lookups only for IPv4 addresses This is the new default since Pi-hole FTL v5.3.2. It should resolve issues with more and more short-lived PE IPv6 addresses seen in a lot of networks.
.br
    ALL - Do the hourly PTR lookups for all addresses This is the same as what we're doing with FTL v5.3(.1). This can create a lot of PTR queries for those with many IPv6 addresses in their networks.
.br
    NONE - Do not do any hourly PTR lookups This means we look host names up exactly once (when we first see a client) and never again. You may miss future changes of host names.
.br
```

See https://github.com/pi-hole/docs/pull/424/files#diff-dbb505ffcfe4d2415d236079fe8fa9fb2a8aef263482fd6182759d9cd27737bf

Reply:
Hmn, unfortunately the line length results in a bit an ugly formatting on a narrow terminal:
```
REFRESH_HOSTNAMES=IPV4|ALL|NONE
    Change  how (and if) hourly PTR requests are made to check for changes in client and upstream server hostnames. The following options are avail‐
able:
    IPV4 - Do the hourly PTR lookups only for IPv4 addresses. This is the new default since Pi-hole FTL v5.3.2. It should resolve issues  with  more
and more short-lived PE IPv6 addresses seen in a lot of networks.
    ALL  - Do the hourly PTR lookups for all addresses. This is the same as what we're doing with FTL v5.3(.1). This can create a lot of PTR queries
for those with many IPv6 addresses in their networks.
    NONE - Do not do any hourly PTR lookups. This means we look host names up exactly once (when we first see a client) and  never  again.  You  may
miss future changes of host names.
```
So far there are no other entries that break this formatting. Often they were split up into multiple lines and the descriptions where generally kept short.

How about:
```
REFRESH_HOSTNAMES=IPV4|ALL|NONE
    Change how (and if) hourly PTR requests are made to check for changes in client and upstream server hostnames:
    IPV4 - Do the hourly PTR lookups only for IPv4 addresses resolving issues in networks with many short-lived PE IPv6 addresses.
    ALL  - Do the hourly PTR lookups for all addresses. This can create a lot of PTR queries in networks with many IPv6 addresses.
    NONE - Don't do hourly PTR lookups. Look up hostnames once (when first seeing a client) and never again. Future hostname changes may be missed.
```
I personally don't feel that it's required to add the details about which version behave how. This is a man-page not a changelog. If I understand correctly this man-page won't ever be seen by a user still using an old version.

Reply:
Fine with me.